cola-tests/src/main/java/com/github/bmsantos/core/cola/utils/ColaUtils.java

```diff
@@ -67,12 +68,24 @@ public static String binaryToOsClass(final String binaryFormat) {
         return result;
     }
 
+    public static String binaryToResource(final String binaryFormat) {
+        return binaryFormat.replace(".", RESOURCE_SEPARATOR);
+    }
+
+    public static String binaryToResourceClass(final String binaryFormat) {
+        String result = binaryToResource(binaryFormat);
+        if (!isClassFile(result)) {
+            result += CLASS_EXT;
+        }
+        return result;
+    }
+
     public static boolean binaryFileExists(final String dir, final String clazz) {
         return isSet(dir) && isSet(clazz) && new File(dir + separator + binaryToOsClass(clazz)).exists();
     }
 
     public static String toOSPath(final String path) {
-        return path.replace("/", separator).replace("\\", separator);
+        return path.replace(RESOURCE_SEPARATOR, separator).replace("\\", separator);
```

Comment:
I don't care all that much, but I wonder about this function's behavior on Windows.